Formula/scrcpy.rb

```diff
@@ -95,3 +98,24 @@ def caveats
     assert_match(/push #{prefix}/, log_content)
   end
 end
+
+__END__
+--- a/app/src/screen.c
++++ b/app/src/screen.c
+@@ -816,7 +816,7 @@ sc_screen_handle_event(struct sc_screen *screen, SDL_Event *event) {
+     bool relative_mode = sc_screen_is_relative_mode(screen);
+
+     switch (event->type) {
+-        case SC_EVENT_SCREEN_INIT_SIZE:
++        case SC_EVENT_SCREEN_INIT_SIZE: {
+             // The initial size is passed via screen->frame_size
+             bool ok = sc_screen_init_size(screen);
+             if (!ok) {
+@@ -824,6 +824,7 @@ sc_screen_handle_event(struct sc_screen *screen, SDL_Event *event) {
+                 return false;
+             }
+             return true;
++        }
+         case SC_EVENT_NEW_FRAME: {
+             bool ok = sc_screen_update_frame(screen);
+             if (!ok) {
```

Comment:
Let's try this. Will upstream if it works.

As for why: https://stackoverflow.com/questions/57742010/error-in-switch-case-a-label-can-only-be-part-of-a-statement-and-a-declaration

Reply:
It works.

Reply:
Fix submitted upstream: https://github.com/Genymobile/scrcpy/pull/3787.